### PR TITLE
doc: suite writing tut: updates following feedback

### PR DESCRIPTION
**Pull request by matthewrmshin**
Follow on from #1839.

### CODE REVIEW

File: doc/rose-rug-suite-writing-tutorial.html [hunk 1]
```diff
@@ -370,10 +370,9 @@ <h3 class="alwayshidden">Add a Build Application - directory</h3>
 mkdir app/fcm_make_navigate
 </pre>
 
-      <p>Create an empty <samp>rose-app.conf</samp> file in the new
-      directory:</p>
+      <p>Create a <samp>rose-app.conf</samp> file in the new directory:</p>
       <pre class="shell">
-touch app/fcm_make_navigate/rose-app.conf
+echo 'mode=fcm_make' &gt;app/fcm_make_navigate/rose-app.conf
```
Comment (matthewrmshin) [inline doc/rose-rug-suite-writing-tutorial.html]: Empty `rose-app.conf` confuses some.

File: doc/rose-rug-suite-writing-tutorial.html [hunk 2]
```diff
@@ -787,6 +787,10 @@ <h3 class="alwayshidden">Visualising the Data</h3>
 
       <p>You'll now get a useful link in your navigate task output. Expect a
       wiggly course!</p>
+
+      <p>Note: If you are viewing the job log via Rose Bush, try switching on
+      Tags mode by clicking on the <kbd>Tags</kbd> at the top right hand corner
+      of the page.</p>
```
Comment (matthewrmshin) [inline doc/rose-rug-suite-writing-tutorial.html]: This is some what related to #1809. I think our version of Jinja2's `urlize` wasn't converting the link properly.

File: doc/rose-rug-suite-writing-tutorial.html [hunk 3]
```diff
@@ -805,9 +809,7 @@ <h3 class="alwayshidden">Adding Alternate Dependency</h3>
       <samp>suite.rc</samp> file:</p>
       <pre class="prettyprint lang-cylc">
         [[[T12]]]
-            graph = """
-                navigate =&gt; take_sun_sight
-            """
+            graph = navigate =&gt; take_sun_sight
```
Comment (matthewrmshin) [inline doc/rose-rug-suite-writing-tutorial.html]: More consistency changes. One liner can be done without the triple quotes.

**Comment by matthewrmshin** [comment]
@arjclark please review.

**Comment by arjclark** [comment]
All looks good to me.

**Pull request merged**